### PR TITLE
Rewrite

### PR DESCRIPTION
**Pull request by adamtherookie**
The rewrite branch now has all the features of the main branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,315 +1,5 @@
...
-
-Jaiden Collins - Former Lead Developer
-
-Alex Carson - Former Lead Developer
-
-Adam Salt - Former Lead Developer
-
-### Please join the Cob:web-Aclevo discord to talk to us and contribute to our projects: https://discord.gg/z3GXvGtuek
-
-* Please be aware that our CDN automatically updates, however it may take up to 2 hours for the cache to clear on the CDN.
```
Comment (ghost) [inline README.md]: This section here should remain on the GitHub readme

File: package.json [hunk 2]
```diff
@@ -1,28 +1,15 @@
...
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "compile": "sass scss/cssheet.scss css/cssheet.css",
-    "minify": "npx postcss css/cssheet.css > css/cssheet.min.css"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/Cobweb-Development/cssheet.git"
+  "version": "0.0.0",
```
Comment (ghost) [inline package.json]: This is version v1.3.0 of CSSheet

File: README.md [hunk 3]
```diff
@@ -1,315 +1,5 @@
...
+# CSSheet
+The official repository for CSSheet! A simple, easy to learn and bootstrap-compatible CSS framework which allows you to make great websites easily!
 
+# How to use
+See our [documentation](#)
```
Comment (ghost) [inline README.md]: There is no documentation.md file, please create one and write out the full documentation before merging,

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
When you have completed the requested changes please contact me on Discord for me to re-review the PR, thank you.

**Comment by TheOceanBreeze** [comment]
Ok to merge.

**Review by TheOceanBreeze — APPROVED** [review]

**Pull request merged**